tests/helpers.py

```diff
@@ -0,0 +1,24 @@
+"""
+Helper functions file for OCS QE
+"""
+import datetime
+
+
+def create_unique_object_name(object_description, object_type):
+    """
+    Creates a unique object name by using the object_description
+    and object_type, as well as the current date/time string.
+    This can be used for any objects such as VMs, disks, clusters etc.
```

Comment:
VMs, disks, cluster etc -> Change to something OCS related ;)

Reply:
Done :)